### ISSUE TITLE
Linux / Mac clique script

### ISSUE THREAD

**Issue by juanfranblanco** [Nethereum/Nethereum.Workbooks#3]
A separate script is needed for either Linux or Mac for clique, and a renamed geth executable for either to distinguish with the other geth.

**Comment by juanfranblanco** [comment]
Parity too, if possible.

**Comment by anegg0** [comment]
This issue is to be closed, right @juanfranblanco ?

**Comment by juanfranblanco** [comment]
No, it needs to be 2 separate scripts pointing to 2 different executables. One Mac the other Linux

**Comment by juanfranblanco** [comment]
For example:
1. LaunchLinux.sh pointing to "linux_geth"
2. LaunchMac.sh pointing to "mac_geth"
3. LaunchWindows.bat pointing to geth.exe

**Comment by juanfranblanco** [comment]
@anegg0 now clique has gone in one of the commits ?

**Comment by juanfranblanco** [comment]
@anegg0  are the workbooks pointing to the new TestChains? shall we close this now?

**Comment by anegg0** [comment]
They are all pointing to Testchains, we can close this.

Gaël Blanchemain

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐
